pydax/_dataset.py

```diff
@@ -43,7 +43,7 @@ def list_all_datasets() -> Dict[str, Tuple]:
     :return: Mapping of available datasets and their versions
     """
 
-    dataset_schema = load_schemata().schemata['dataset_schema'].export_schema('datasets')
+    dataset_schema = get_schemata().schemata['datasets'].export_schema('datasets')
```

Comment:
`export_schema` should have already produced a copy. But it is good advice that we should avoid using `get_schemata` because that causes potential accidental change and causes very little performance concern.